### ISSUE TITLE
ncm-fmonagent: Archive and remove from repository

### ISSUE THREAD

**Issue by jrha** [quattor/configuration-modules-core#1053]
See #1027.

**Comment by jrha** [comment]
@ned21 can this component be archived now?

**Comment by ned21** [comment]
Looks like it's still installed on all our RH6 & RH7 hosts. I think the plan is to finally kill it as part of RH8 upgrade. @aka7 ?

**Comment by aka7** [comment]
yes correct, will be killed off in rhel8 but still needed on rhel6 and 7 it seems.  I will try to find out what's actually using it on rhel6/7 to see if we can kill it off, but unfortunately as it stands we still need it.

**Comment by jrha** [comment]
Okay thanks.
